vaccine.py

```diff
@@ -1,50 +1,134 @@
-'''
-This is a python script that requires you have python installed, or in a cloud environment.
-
+#!/usr/bin/env python3
+#
+"""
 This script scrapes the CVS website looking for vaccine appointments in the cities you list.
-To update for your area, update the locations marked with ### below.
 
-If you receive an error that says something is not install, type
+Usage: ./vaccine.py --state [STATE] --cities [CITIES] --duration [HOURS]
+
+Required arguments:
 
-pip install beepy
+    STATE       the two-letter abbreviation for your state (e.g., IL, CA, etc.)
+    CITIES      a JSON file containing an array of strings, which list all the cities
+                you are willing to travel to for a vaccination.
 
-in your terminal.
-'''
+Optional (default will be used if not specified):
 
+    HOURS       how long should this script run for (default: 3)
+    INTERVAL    how often to run the check in minutes (default: 10)
 
+"""
 
 import requests
 import time
-import beepy
 
+HAS_SOUND = True
+try:
+    import beepy
```

Comment:
On Linux (Ubuntu 20.4) importing `beepy` fails with some error or other around the audio library: life being short as it is, I decided not to worry too much about it, and use this workaround instead.